src/components/Auth/assets/FaceTecStrings.tsx

```diff
@@ -0,0 +1,177 @@
+import { TranslateCallback } from '~types/locales'
+
+const FEEDBACK = {
+  center_face: 'auth_capture.feedback.center_face',
+  face_not_found: 'auth_capture.feedback.face_not_found',
+  move_back: 'auth_capture.feedback.move_back',
+  move_close: 'auth_capture.feedback.move_close',
+  eye_level: 'auth_capture.feedback.eye_level',
+  not_looking_straight_ahead:
+    'auth_capture.feedback.not_looking_straight_ahead',
+  head_not_upright: 'auth_capture.feedback.head_not_upright',
+  steady: 'auth_capture.feedback.steady',
+  move_closer: 'auth_capture.feedback.move_closer',
+  even_lighting: 'auth_capture.feedback.even_lighting',
+}
+
+const PRESESSION_FEEDBACK = {
+  center_face: 'auth_capture_start.feedback.center_face',
+  head_not_upright: 'auth_capture_start.feedback.head_not_upright',
+  not_looking_straight: 'auth_capture_start.feedback.not_looking_straight',
+  steady_count_1: 'auth_capture_start.feedback.steady_count_1',
+  steady_count_2: 'auth_capture_start.feedback.steady_count_2',
+  steady_count_3: 'auth_capture_start.feedback.steady_count_3',
+  remove_sunglasses: 'auth_capture_start.feedback.remove_sunglasses',
+  neutral_expression: 'auth_capture_start.feedback.neutral_expression',
+  conditions_too_bright: 'auth_capture_start.feedback.conditions_too_bright',
+  conditions_too_dark: 'auth_capture_start.feedback.conditions_too_dark',
+}
+
+export const FaceTecStrings = (translate: TranslateCallback) => ({
```

Comment:
Sorry I'm lazy here, but the best way should be to fully declare the return type for better consumption.

```suggestion
export const FaceTecStrings = (translate: TranslateCallback): Record<string, string> => ({
```

Reply:
As I can see, still no `translate` validations here

Reply:
Sorry, I think I'm confused. `translate` validations? referring to https://github.com/onfido/onfido-sdk-ui/blob/b17e9f2aff972d422c61b13496647700abf57600/src/components/utils/localesMapping.ts#L3-L16 ?

Reply:
It was a suggestion before you converted this into TypeScript, so I think it's valid now.